README.md

```diff
@@ -1,2 +1,9 @@
-# appcircle-change-java-version-component
-The Change Java Version component allows switching between different Java versions in Appcircle workflow.
\ No newline at end of file
+# Appcircle Change Java Version
+
+This step switches to the selected Java version during the build process. 
+
+If your runner is self-hosted, ensure that the selected Java version is available in your environment. If not, you must install the required version before this step.
+
+## Required Input Variables
+
+- `$AC_SELECTED_JAVA_VERSION`: Select the Java version to switch to.
```

Comment:
@boztopuz I deliberately did not extend this explanation, I kept it simple and understandable.
Did you think it was incomprehensible?
If so, I can improve it further, but I did not find the suggested change correct. The reasons are:

### `Change`:
This improvement will come with the following default variable. Here, we want it to do "Select", not "Change"

### `in the environment`:
This component does not only change the environment, it also changes the system Java version.

### `to the selected Java version`
I avoided repeating words in the same sentence, so I deliberately did not add this, but I will improve it a little more.

How to use the following:
"Select the Java version to switch to. If the selected Java version is not found in the runner, it will throw an error."

Reply:
The last change you mentioned is okey